rules/android/android_binary.bzl

```diff
@@ -82,6 +82,7 @@ def android_binary(
             baseline = lint_baseline,
             lint_config = lint_options.get("config", None),
             deps = kotlin_library_deps,
+            custom_lint_rule_targets = lint_options.get("customLintTargets", default = []),
```

Comment:
Can we name this `lint_checks` to be consistent with what we are passing to `project.xml` instead? Both here and in `lint_options`?

Also should we apply this to `android_library` too?

Reply:
yes and yes